src/tests/banners/bannerSelection.tsx

```diff
@@ -75,29 +143,29 @@ export const selectBannerTest = (
     targeting: BannerTargeting,
     pageTracking: BannerPageTracking,
     baseUrl: string,
-): Promise<BannerTestSelection | null> => {
-    return redeployedSinceLastClosed(targeting).then(hasRedeployed => {
-        if (hasRedeployed) {
-            const tests = [AusMomentContributionsBanner];
-            const testToRun = tests.find(
-                test =>
-                    targeting.alreadyVisitedCount >= test.minPageViews &&
-                    !targeting.shouldHideReaderRevenue &&
-                    !targeting.isPaidContent &&
-                    test.canRun(targeting, pageTracking),
-            );
-
-            if (testToRun) {
-                const variant = testToRun.variants[0]; // TODO - use mvt
+): BannerTestSelection | null => {
+    const tests = [AusMomentContributionsBanner, SubscriptionsBanner, WeeklyBanner];
+    tests.find(test => {
+        redeployedSinceLastClosed(targeting, test.bannerType).then(hasRedeployed => {
```

Comment:
I would think about checking all the other targeting data before the redeploy dates as they don't involve an http request but happy to leave it as is for this PR and think about refactoring later.